pkg/controller/controller.go

```diff
@@ -182,7 +187,7 @@ func GetCommandHandlers(ctx *context.Provider, opts ...Opt) ([]command.Handler,
 	}
 
 	// verifiable command operation
-	verifiablecmd, err := verifiable.New(ctx)
+	verifiablecmd, err := verifiable.New(ctx, cmdOpts.kms, verifiabledoc.NewDIDKeyResolver(cmdOpts.vdri))
```

Comment:
Since command is common, can we just pass `ctx`(with kms and vdri bundled in already) here and create DIDKeyResolver inside command.go ?

Reply:
done